selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/theia/TheiaIde.java

```diff
@@ -116,4 +116,9 @@ public void closeAboutDialog() {
   public void waitAboutDialogContains(String expectedText) {
     seleniumWebDriverHelper.waitTextContains(aboutDialogContent, expectedText);
   }
+
+  public void switchToIdeFrame() {
```

Comment:
To have browser window closed after test has been executed, we need to add pre-destroy method in the class:
```
  @PreDestroy
  public void close() {
    seleniumWebDriver.quit();
  }
```